app/models/user.rb

```diff
@@ -85,6 +84,18 @@ def role_changed_to_editor?
     role_changed_to_editor
   end
 
+  def clear_remotely_signed_out!
+    # rubocop:disable Rails/SkipsModelValidations
+    update_attribute(:remotely_signed_out, false)
```

Comment:
not sure why update_attribute is used here - maybe to make sure it happens despite the state the user is in?

Reply:
I don't think we need the `remotely_signed_out!` stuff? The existing stuff is reliant on the GOV.UK Signon servers calling an API exposed by gds_sso gem.